### PR TITLE
merge dev into master—oct 2023 edition

### PR DESCRIPTION
**Pull request by rtapella**

### CODE REVIEW

File: docs/source/system_reference_guide/jobsui.ipynb [hunk 1]
```diff
@@ -15,11 +15,11 @@
    "id": "db3a214d-4c06-426e-a00c-1ffc8fec73b2",
    "metadata": {},
    "source": [
-    "## Access\n",
+    "## Access the Jobs UI\n",
     "\n",
-    "1. From your workspace click on the __View & Submit Jobs__ card on the launcher tab.\n",
+    "1. From your workspace click on the __View & Submit Jobs__ card on the >auncher tab.\n",
```
Comment (emileten) [inline docs/source/system_reference_guide/jobsui.ipynb:20]:
```suggestion
    "1. From your workspace click on the __View & Submit Jobs__ card on the Launcher tab.\n",
```

File: docs/source/system_reference_guide/share_data.ipynb
Comment (emileten) [inline docs/source/system_reference_guide/share_data.ipynb:1]: I find that there is significant redundancy between this and https://github.com/MAAP-Project/maap-documentation/blob/develop/docs/source/system_reference_guide/faq/persistent_files.ipynb. Maybe we could create a ticket to centralize the information and have the faq answer just point to it.

**Review by emileten — CHANGES_REQUESTED** [review]
A few small changes requested that are directly related to the changes in this PR, a few questions that could open new tickets, and before this get merged we should add this PR I just created that fixes small things I discovered in the technical tutorials https://github.com/MAAP-Project/maap-documentation/pull/346.

File: docs/source/system_reference_guide/share_data.ipynb
Comment (wildintellect) [reply docs/source/system_reference_guide/share_data.ipynb:1]: @emileten Sure you can make such a ticket, this is not a blocker for this PR.

File: docs/source/system_reference_guide/algorithm_registration.ipynb
[the remark this answers is not in the document]
Comment (rtapella) [reply docs/source/system_reference_guide/algorithm_registration.ipynb:1]: It is very similar. At some point I would expect to use the "system reference guide" as the generic/main doc for this, and the getting started guide will be specific to the demo algorithm.

File: docs/source/system_reference_guide/share_data.ipynb
Comment (rtapella) [reply docs/source/system_reference_guide/share_data.ipynb:1]: Yes I agree @emileten

File: docs/source/getting_started/running_at_scale.ipynb [hunk 2]
```diff
@@ -78,8 +78,7 @@
     "### The Jobs UI\n",
     "MAAP is configured to run up to 4,000 concurrent jobs. There are two additional ways to run a job: via the Jobs UI in the Launcher, or via a call to the maap-py Python library.\n",
     "\n",
-    "The Jobs UI will let you run and monitor jobs easily. You can find full documentation in the system reference guide for [the Jobs UI](../system_reference_guide/ade_custom_extensions/jobsui.ipynb).\n",
-    "You can also find specific documentation on [how to submit jobs](../system_reference_guide/faq/job_submit.ipynb) and [how to monitor jobs](../system_reference_guide/faq/job_check_status.ipynb) in the System Reference Guide FAQs.\n",
+    "The Jobs UI will let you run and monitor jobs easily. You can find full documentation in the system reference guide for [the Jobs UI](../system_reference_guide/jobsui.ipynb) or [using maap-py](../system_reference.rst/jobs_maappy.ipynb) with Python in the System Reference Guide FAQs.\n",
```
[the remark this answers is not in the document]
Comment (rtapella) [reply docs/source/getting_started/running_at_scale.ipynb:81]: These changes are good but I am struggling to get git to commit the suggestions...

File: docs/source/system_reference_guide/create_project.ipynb
[the remark this answers is not in the document]
Comment (grallewellyn) [reply docs/source/system_reference_guide/create_project.ipynb:1]: I agree we should just remove from ADE. Issue to remove here: https://github.com/MAAP-Project/Community/issues/798

File: docs/source/getting_started/writing_code.ipynb [hunk 3]
```diff
@@ -53,28 +53,38 @@
...
     "\n",
-    "5. If you want to make changes to the code and have your own copy of it to register, Clone the code into a public repository in Github or in MAAP Gitlab."
+    "4. If you want to make changes to the code and have your own copy of it to register, clone the code into a public repository in Github or in MAAP Gitlab."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Clone a Test Algorithm with GitHub\n",
```
[the remark this answers is not in the document]
Comment (grallewellyn) [reply docs/source/getting_started/writing_code.ipynb:80]: I think we should keep "Clone a Repository with GitHub" section, and I think we can do without "Clone a Test Algorithm with GitHub" as long as we have a note at the bottom of "Clone a Repository with GitHub" saying follow these steps for cloning a test algorithm

File: docs/source/system_reference_guide/jobsui.ipynb (hunk 1 above)
Comment (rtapella) [reply docs/source/system_reference_guide/jobsui.ipynb:20]: in https://github.com/MAAP-Project/maap-documentation/pull/347

File: docs/source/getting_started/writing_code.ipynb [hunk 4]
```diff
@@ -53,28 +53,38 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Clone a Test Algorithm with GitHub\n",
+    "### Clone a Repository with GitHub\n",
     "\n",
-    "Any git repository online can be used to host code. In order to register your algorithm with the DPS, the repo must be public. Here is an example algorithm you can use for this getting started guide:\n",
+    "Any git repository online can be used to host code. Here is an example algorithm you can use for this getting started guide:\n",
```
[the remark this answers is not in the document]
Comment (rtapella) [reply docs/source/getting_started/writing_code.ipynb:58]: in https://github.com/MAAP-Project/maap-documentation/pull/347

File: docs/source/getting_started/writing_code.ipynb [hunk 5]
```diff
@@ -53,28 +53,38 @@
...
-    "5. If you want to make changes to the code and have your own copy of it to register, Clone the code into a public repository in Github or in MAAP Gitlab."
+    "4. If you want to make changes to the code and have your own copy of it to register, clone the code into a public repository in Github or in MAAP Gitlab."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Clone a Test Algorithm with GitHub\n",
+    "In order to register your algorithm with the DPS, the repo must be public. Here is an example algorithm you can use for this getting started guide:\n",
```
[the remark this answers is not in the document]
Comment (rtapella) [reply docs/source/getting_started/writing_code.ipynb:81]: in https://github.com/MAAP-Project/maap-documentation/pull/347

File: docs/source/getting_started/writing_code.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (rtapella) [reply docs/source/getting_started/writing_code.ipynb:80]: in https://github.com/MAAP-Project/maap-documentation/pull/347

File: docs/source/getting_started/running_at_scale.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (rtapella) [reply docs/source/getting_started/running_at_scale.ipynb:81]: fixed in https://github.com/MAAP-Project/maap-documentation/pull/347

**Comment by rtapella** [comment]
@emileten how does it look now?

File: docs/source/system_reference_guide/share_data.ipynb
Comment (emileten) [reply docs/source/system_reference_guide/share_data.ipynb:1]: in #348

**Review by emileten — APPROVED** [review]

**Comment by emileten** [comment]
Great, thanks for the changes @rtapella ! Feel free to merge.

**Pull request merged**